src/wallet/wallet.h

```diff
@@ -73,6 +73,11 @@ static const unsigned int DEFAULT_TX_CONFIRM_TARGET = 6;
 static const bool DEFAULT_WALLET_RBF = false;
 static const bool DEFAULT_WALLETBROADCAST = true;
 static const bool DEFAULT_DISABLE_WALLET = false;
+//! -maxtxfee default
+constexpr CAmount DEFAULT_TRANSACTION_MAXFEE = COIN / 10;
+
+/** Absolute maximum transaction fee (in satoshis) used by wallet */
+extern CAmount g_max_tx_fee;
```

Comment:
Please no more globals :hand:  :weary: 

This will just force future refactoring changes to make it a wallet member, when we allow per-wallet settings.

See `CWallet::m_min_fee`  and (wallet: Make fee settings to be non-static members #12909)  for inspiration

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15778#discussion_r274436217

> See CWallet::m_min_fee and (wallet: Make fee settings to be non-static members #12909) for inspiration

A suggestion would be to call the setting `m_default_max_tx_fee`, since similar to [`m_default_address_type`/`m_default_change_type`](https://github.com/bitcoin/bitcoin/blob/bb68abe784b9fd49bddf38105d142a31ef0f9e77/src/wallet/wallet.h#L984-L985) it's a default preference and something that could be overridden for individual wallet operations.

Reply:
Yes, I guess that makes sense and fits with what I originally wrote here: https://github.com/bitcoin/bitcoin/issues/13044. It just seems strange to me to have options which (currently) apply to all wallets stored in each individual `CWallet` (see also `fBroadcastTransactions` which is a `CWallet` member, but I don't think makes sense as a per-wallet config).

I'll move `max_tx_fee` to live alongside the other wallet member fee settings.

Reply:
fixed